naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/schema/NaptimeResourceUtils.scala

```diff
@@ -191,4 +199,45 @@ object NaptimeResourceUtils extends StrictLogging {
       }
       .toSet
     }
+
+  private[schema] def interpolate(
+      argumentValue: String,
+      variableNameToInterpolatedIds: Map[String, List[String]]): List[String] = {
+
+    // Note: For simplicity, we do not support cross product of ids. Instead, we
```

Comment:
FYI, here is the logic we are going with when interpolating multiple variables. I don't think right now anyone interpolates multiple variables and multiple ids per variable, so this impl should not affect any existing behaviors.